project-novica/demo/src/main/java/com/example/demo/controller/HelloController.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.controller;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @RequestMapping("/helloworld")
```

Comment:
By convention, we should use **hyphenated style** when naming routes, instead of `/HelloWorld` change it to `/hello-world`